### ISSUE TITLE
Error in executing ResNet 50

### ISSUE THREAD

**Issue by novelmartis** [vlfeat/matconvnet#509]
I get the following error when I try to load resnet-50-dag :

No public property epsilon exists for class dagnn.BatchNorm.

Error in dagnn.Layer/load (line 185)
        obj.(f) = s.(f) ;

Error in dagnn.DagNN.loadobj (line 27)
    block.load(struct(s.layers(l).block)) ;

Error in untitled (line 4)
net = dagnn.DagNN.loadobj(load('imagenet-resnet-50-dag.mat')) ;

Could someone help me understand what's going on?

**Comment by vedaldi** [comment]
Hi, can you confirm that you are using the latest version of MCN to load this model?

**Comment by novelmartis** [comment]
Hi,

I just upgraded from beta-18 to beta-19, and now even the GoogLeNet isn't
working (DAG) [Using the code mentioned on the quick start page]

The error is:

Attempt to execute SCRIPT vl_nnconv as a function:
/Users/isush4u/matconvnet-1.0-beta18/matlab/vl_nnconv.m

Error in dagnn.Conv/forward (line 11)
      outputs{1} = vl_nnconv(...

Error in dagnn.Layer/forwardAdvanced (line 85)
      outputs = obj.forward(inputs, {net.params(par).value}) ;

Error in dagnn.DagNN/eval (line 88)
  obj.layers(l).block.forwardAdvanced(obj.layers(l)) ;

Error in untitled3 (line 11)
net.eval({'data', im_}) ;

Thanks

On Tue, Apr 19, 2016 at 10:49 AM, Andrea Vedaldi <notifications@github.com>
wrote:

> Hi, can you confirm that you are using the latest version of MCN to load
> this model?
>
> > On 19 Apr 2016, at 09:45, Sushrut Thorat <notifications@github.com>
> wrote:
> >
> > I get the following error when I try to load resnet-50-dag :
> >
> > No public property epsilon exists for class dagnn.BatchNorm.
> >
> > Error in dagnn.Layer/load (line 185)
> > obj.(f) = s.(f) ;
> >
> > Error in dagnn.DagNN.loadobj (line 27)
> > block.load(struct(s.layers(l).block)) ;
> >
> > Error in untitled (line 4)
> > net = dagnn.DagNN.loadobj(load('imagenet-resnet-50-dag.mat')) ;
> >
> > Could someone help me understand what's going on?
> >
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly or view it on GitHub <
> https://github.com/vlfeat/matconvnet/issues/509>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/vlfeat/matconvnet/issues/509#issuecomment-211805775>
>



-- 
Sushrut Thorat
Masters student in Cognitive Sciences
UTrento, Italy
+39 3803764733
http://novelmartis.github.io

**Comment by novelmartis** [comment]
Hi,

I figured out the problem. I had not added the "matlab" folder into
Matlab's default path.. Sorry for the trouble. It is all functional now :)

On Tue, Apr 19, 2016 at 10:49 AM, Andrea Vedaldi <notifications@github.com>
wrote:

> Hi, can you confirm that you are using the latest version of MCN to load
> this model?
>
> > On 19 Apr 2016, at 09:45, Sushrut Thorat <notifications@github.com>
> wrote:
> >
> > I get the following error when I try to load resnet-50-dag :
> >
> > No public property epsilon exists for class dagnn.BatchNorm.
> >
> > Error in dagnn.Layer/load (line 185)
> > obj.(f) = s.(f) ;
> >
> > Error in dagnn.DagNN.loadobj (line 27)
> > block.load(struct(s.layers(l).block)) ;
> >
> > Error in untitled (line 4)
> > net = dagnn.DagNN.loadobj(load('imagenet-resnet-50-dag.mat')) ;
> >
> > Could someone help me understand what's going on?
> >
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly or view it on GitHub <
> https://github.com/vlfeat/matconvnet/issues/509>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/vlfeat/matconvnet/issues/509#issuecomment-211805775>
>



-- 
Sushrut Thorat
Masters student in Cognitive Sciences
UTrento, Italy
+39 3803764733
http://novelmartis.github.io

**Comment by lenck** [comment]
Great to hear that! :)
